content/documentation/staging/faq/distributed-tracing/see-url-about-modal.adoc

```diff
@@ -1,12 +0,0 @@
----
-title: I can’t see the link to Tracing or traces tab in service details
-weight: 70
----
-:icons: font
-:imagesdir: /images/faq/tracing/
-
-To see Jaeger integration (both link to browse traces and traces tabs in service details) Kiali needs to know the URL where Jaeger is running under. In order for Kiali users to know which URL Kiali users, check the About menu, as it follows:
-
-image::about_menu.png[About menu]
-
-image::about.png[About modal]
```

Comment:
This is a FAQ to debug if the user can't see the tab traces

Reply:
I've put back an entry about missing URL, but it's quite different from how it was before. The Traces tab should always be visible unless it's explicitly disabled. Note that I mentioned the about menu in the section bout configuring the URL field.